### PR TITLE
Sales-Assistant-Final PR

### PR DESCRIPTION
**Pull request by KeinTung**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [ x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [ x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [ x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x ] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [ x] Select a beautiful and meaningful teaser image for your blog
- [x ] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Review by zhihuifanqiechaodan — DISMISSED** [review]

**Review by qhduan — DISMISSED** [review]

**Comment by KeinTung** [comment]
Hi 高宇：
I resolved a ependency issue and passed the CI/CD.  Please help me approve
this blog again.  Really appreciate it!
https://github.com/wechaty/wechaty.js.org/pull/1477

Bests, Kevin Tung

**Review by qhduan — APPROVED** [review]

**Review by zhihuifanqiechaodan — APPROVED** [review]

**Comment by KeinTung** [comment]
@qhduan @lijiarui @huan 請幫我 approve 一下，謝謝

File: package.json [hunk 1]
```diff
@@ -74,5 +74,10 @@
     "scripts": {
       "pre-push": "npx git-scripts-pre-push"
     }
+  },
+  "dependencies": {
```
Comment (lijiarui) [inline package.json:78]: We should not add any irrelevant files in this PR.

I don't think we should add other packages when we add a blog. So please remove this file, thanks!

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (KeinTung) [reply package.json:78]: https://github.com/wechaty/wechaty.js.org/issues/1478
there is a dependency issue in wechaty.org.js recently, so this modification is necessary

**Review by lijiarui — APPROVED** [review]
Good job! Cheers!

**Comment by KeinTung** [comment]
@huan: 
Hi!  Please have a review on my project, really appreciate it!

**Review by huan — APPROVED** [review]
LGTM

Thank you very much for your great work, I hope this project can be open-sourced and can be keeping improving in the future!

**Comment by lijiarui** [comment]
Thank you very much for your contribution!

## You are welcome to join [Wechaty Contributor Program](https://wechaty.js.org/docs/contributor-program/)

### 1. Join Wechaty Organization

> You've invited KevinTung to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

I have invited you to join our Wechaty GitHub Organization, please accept it by following the above message.

### 2. Update Your Wechaty Contributor Profile

1. Please open [Contributor Hall of Fame](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0) and add yourself to the end of the list, so that other contributors will know you better!
1. Please add yourself to our Website [Contributors](https://wechaty.js.org/contributors/) by [creating a PR](https://github.com/wechaty/wechaty.js.org/tree/master/jekyll/_contributors) and refer to this PR link as well.

### 3. Join The Contributor Only WeChat Room

We also have a WeChat room for contributors only which can discuss Wechaty at a deeper level, you are welcome to join and if you are interested.

Please add my: wechat: _**ruirui_0914**_  and send this pr link to me. I will invite you into Wechaty Contributor Room

Link to https://github.com/wechaty/PMC/issues/16

Cheers!

**Pull request merged**